### PR TITLE
[NETBEANS-54] Module Review Apache License modules

### PR DESCRIPTION
**Pull request by 38leinaD**
Fix emilianbold#197

-no external library
-checked Rat report: added license header to
templates/src/org/netbeans/modules/templates/resources/userprop.txt
-skimmed through module, did not notice additional problems

### CODE REVIEW

**Comment by emilianbold** [comment]
See comment on #197. Make a patch against nbbuild/build.xml

**Comment by emilianbold** [comment]
This PR should only have the exclusion. The file remains without a license header since it generates user files.

**Comment by 38leinaD** [comment]
that makes sense; i updated the pull request.

**Comment by emilianbold** [comment]
> This branch has conflicts that must be resolved

**Comment by 38leinaD** [comment]
resolved

File: nbbuild/build.xml [hunk 1]
```diff
@@ -1968,7 +1968,8 @@ It is possible to use -Ddebug.port=3234 -Ddebug.pause=y to start the system in d
                 <exclude name="diff/test/unit/src/org/netbeans/modules/diff/builtin/visualizer/data/**" /> <!--test data-->
                 <exclude name="utilities/test/unit/src/org/netbeans/modules/openfile/resources/recent_files/*" /> <!--test data-->
                 <exclude name="**/*.url" /> <!--no degree of creativity-->
-		<exclude name="spi.navigator/test/unit/src/org/netbeans/modules/navigator/resources/lastsel/file.lastsel_mime1" /> <!--empty file-->    
+		            <exclude name="spi.navigator/test/unit/src/org/netbeans/modules/navigator/resources/lastsel/file.lastsel_mime1" /> <!--empty file-->
```
Comment (emilianbold) [inline nbbuild/build.xml]: extra tab?

**Review by emilianbold — CHANGES_REQUESTED** [review]

File: nbbuild/build.xml (hunk 1 above)
Comment (38leinaD) [reply nbbuild/build.xml]: ups; person that created spi.navigator-entry used tabs. my editor showed it missaligned and thus i added more spaces. fixed it.

**Review by emilianbold — APPROVED** [review]

**Pull request merged**